library/blocks/in-world-recipes/src/main/java/org/quiltmc/qsl/inworldrecipes/mixin/PickaxeItemMixin.java

```diff
@@ -0,0 +1,32 @@
+package org.quiltmc.qsl.inworldrecipes.mixin;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemUsageContext;
+import net.minecraft.item.PickaxeItem;
+import net.minecraft.util.ActionResult;
+import net.minecraft.world.World;
+import org.quiltmc.qsl.inworldrecipes.api.InWorldRecipeRegistries;
+import org.spongepowered.asm.mixin.Mixin;
+
+@Mixin(PickaxeItem.class)
+public abstract class PickaxeItemMixin extends Item {
+	public PickaxeItemMixin() {
+		super(new Settings());
+		throw new AssertionError();
+	}
+
+	@Override
+	public ActionResult useOnBlock(ItemUsageContext context) {
```

Comment:
overriding methods in mixins is not recommended. I think the ideal path is to inject in a supermixin (mixin to Item) with an empty body and override that handler method in a submixin (this mixin here).